abr_control/controllers/path_planners/dmp_filter.py

```diff
@@ -0,0 +1,126 @@
+"""
+Running operational space control with a PyGame display, and using the pydmps
+library to specify a trajectory for the end-effector to follow, in
+this case, a circle.
+
+To install the pydmps library, clone https://github.com/studywolf/pydmps
+and run 'python setup.py develop'
+"""
+import numpy as np
+
+try:
+    import pydmps
+except ImportError:
+    print('\npydmps library required, see ' +
+          'https://github.com/studywolf/pydmps\n')
+
+import scipy.interpolate
+
+
+class dmpFilter():
```

Comment:
dmp path planner

Reply:
i think this would be good to have, but renamed like `bell_shaped_velocity_profile` or somesuch